Makefile

```diff
@@ -446,7 +446,7 @@ $(DDIR):
 	@mkdir $(DDIR)
 
 $(ODIR)/%.o: $(SRC_DIR)/%.cpp
-	$(CXX) $(DEFINES) $(CXXFLAGS) -c $< -o $@
+	$(CXX) $(CPPFLAGS) $(DEFINES) $(CXXFLAGS) -c $< -o $@
```

Comment:
It allows passing additional flags, which are typically found in CPPFLAGS.
E.g. Debian passes -D_FORTIFY_SOURCE=2 via CPPFLAGS, which would be ignored without this change.